schema/types/labels.go

```diff
@@ -2,9 +2,16 @@ package types
 
 import (
 	"encoding/json"
-	"errors"
+	"fmt"
+	"sort"
 )
 
+var ValidOSArch = map[string][]string{
+	"linux":   []string{"amd64", "i386"}, // what about x86_64, i686, other platforms?
```

Comment:
@mpasternacki Shouldn't we be providing a common interface that is invariant to the whims of hardware developers? Mapping amd64 to x86_64 is trivial and makes cross-platform understanding a lot more straightforward.